packages/video_player/video_player_avfoundation/example/ios/RunnerTests/VideoPlayerTests.m

```diff
@@ -121,6 +122,41 @@ - (void)testHLSControls {
   XCTAssertEqualWithAccuracy([videoInitialization[@"duration"] intValue], 4000, 200);
 }
 
+- (void)testTransformFix {
```

Comment:
There are a lot of paths through that method, so I would expect more tests than this. In particular, we should at the very least have a test covering each of the cases we've seen in the wild of original transforms that were incorrect, and validate that we get the expected output. That will prevent us from continuing with a cycle of people finding new problematic videos, adjusting the tranform fixup code without knowing about all of the previous cases, accidentally regressing one of those, etc.

Reply:
I agreed, but that's a lot as you said. We can add them in separate PRs IMO.

Reply:
You're rewriting the method; adding more than one path through a completely rewritten method as part of testing is not something that should be done later. This code you are changing in this PR should be adequately tested in this PR.

Reply:
Please re-read my other comment thread. I'm not proposing that you add more videos.

Reply:
Now all orientations are under the test.